src/main/kotlin/com/atlassian/performance/tools/virtualusers/api/EntryPoint.kt

```diff
@@ -28,12 +33,22 @@ class Application {
     }
 
     private fun run(args: Array<String>) {
-        val options = VirtualUserOptions.Parser().parse(args)
-        @Suppress("DEPRECATION")
-        if (options.help) {
-            options.printHelp()
-        } else {
-            LoadTest(options, RestUserGenerator()).run()
-        }
+        val options: VirtualUserOptions = VirtualUserOptions.Parser().parse(args)
+
+        setLoggingConfiguration(options)
+
+        if (options.help) options.printHelp()
```

Comment:
You have removed the `@Suppress("DEPRECATION")` which means there is one more warning here. Please revert the change unless there is some specific reason. If so please explain.

Reply:
Also please revert back the `if-else` with braces. We want to avoid problems like this one: https://stackoverflow.com/questions/8020228/is-it-ok-if-i-omit-curly-braces-in-java

Reply:
The main reason why I changed it was simply to reduce lines of code.
Before 6 now 2 it is more readable, but I don't have any strong opinion why I shouldn't revert those lines, so Sure I will revert.

Reply:
Less lines is not always more readable.